### PR TITLE
Added structure for global repository

### PR DESCRIPTION
**Pull request by Itisha2987**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ __pycache__/
 *.pyo
 knowledge_graph.png
 *.png
+*.json
```
Comment (Itisha2987) [inline .gitignore:6]: Remove end of the line error.

File: util.py [hunk 2]
```diff
@@ -125,3 +129,16 @@ def get_intersection_set_of_recognitions(recognitions, possible_recognitions):
...
 
     return recognitions
+
+
+def get_most_similar_element(new_object):
+    objects = get_all_objects()
+    objects = convert_data_to_objects(objects)
+    highest_similarity = 0
+    for element in objects:
+        similarity = element.get_similarity_index(new_object)
```
Comment (Itisha2987) [inline util.py]: Add TODO for Get_Similarity_index method.

File: util.py [hunk 3]
```diff
@@ -125,3 +129,16 @@ def get_intersection_set_of_recognitions(recognitions, possible_recognitions):
...
+        if similarity > highest_similarity:
+            highest_similarity = similarity
+            similar_object = element
+    if highest_similarity > 0.75:
+        return similar_object
```
Comment (Itisha2987) [inline util.py]: This will just return a single element with highest similarity.

File: util.py [hunk 4]
```diff
@@ -125,3 +129,16 @@ def get_intersection_set_of_recognitions(recognitions, possible_recognitions):
...
     return recognitions
+
+
+def get_most_similar_element(new_object):
+    objects = get_all_objects()
+    objects = convert_data_to_objects(objects)
+    highest_similarity = 0
+    for element in objects:
+        similarity = element.get_similarity_index(new_object)
+        if similarity > highest_similarity:
```
Comment (Itisha2987) [inline util.py]: We decided to print all the objects with similarity Index greater than 0.75

File: similarity.py [hunk 5]
```diff
@@ -8,12 +8,13 @@ def get_similarity_index(old_object, new_object):
 	return 0.8
 
 
-def get_most_similar_element(new_object):
+def get_most_similar_element(new_object_name):
     objects = data['objects']
+    new_object = get_object_data(new_object_name)
```
Comment (Itisha2987) [inline similarity.py]: get_object_data_through_name is the method name :p

Comment (karan-modh) [reply similarity.py]: already fixed

**Comment by karan-modh** [comment]
Looks Good to me.
Merging into main

**Pull request merged**